ui.py

```diff
@@ -48,7 +48,20 @@
 import logging
 LookingGlassAddonLogger = logging.getLogger('Alice/LG')
 
-
+# ---------------- STRING CACHE --------------------------
+# see: https://blender.stackexchange.com/questions/299978/how-to-fix-unicodedecodeerror
+_alicelg_string_cache = {}
+def intern_enum_items(items):
+	def intern_string(s):
+		if not isinstance(s, str):
+			return s
+		global _alicelg_string_cache
+		if s not in _alicelg_string_cache:
+			_alicelg_string_cache[s] = s
+			print(len(_alicelg_string_cache))
+			# print(s)
+		return _alicelg_string_cache[s]
+	return [tuple(intern_string(s) for s in item) for item in items]
```

Comment:
For a more consistent code structuring, I would like to see this in the globals.py. Could you move it at the end of the globals.py under the "global add-on functions" section? thanks a lot!

Reply:
I moved this function to globals.py, but I can't declare it as a staticmethod because of the scope of `_alicelg_string_cache`.

If `_alicelg_string_cache` is a class variable and `intern_enum_items` is a classmethod, it may not work as expected when I tested before.